controllers/sriovoperatorconfig_controller.go

```diff
@@ -119,6 +122,12 @@ func (r *SriovOperatorConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 		return reconcile.Result{}, err
 	}
 
+	if utils.ClusterType == "openshift" {
```

Comment:
"openshift" ClusterType is appearing in several locations, consider adding a constant to utils

Reply:
fixed